python/lsst/cp/pipe/ptc.py

```diff
@@ -145,6 +145,11 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Sigma cut for outlier rejection in PTC.",
         default=5.0,
     )
+    sigmaCutAfwMathStatsControl = pexConfig.Field(
+        dtype=float,
+        doc="Sigma cut for afwMatch.StatisticsControl()",
+        default=3.0,
```

Comment:
Can we change this to 5.5 please?

Reply:
On `None` and use the default if not set